content/docs/api-reference/open.md

```diff
@@ -8,7 +8,8 @@ def open(path: str,
          rev: str = None,
          remote: str = None,
          mode: str = "r",
-         encoding: str = None)
+         encoding: str = None,
+         config: dict = None)
```

Comment:
Side-rant, but the formatting of python code in our docs still grinds my gears. I'm sure other people used to black are scratching their eyes at this spacing and lots of single-quotes elsewhere. No action needed, but just wanted to complain 🙂